test/arrays/ferguson/semantic-examples/valgrind-futures/4-return-alias-local.chpl

```diff
@@ -1,7 +1,7 @@
 use samples;
 proc f() {
   var A:[1..4] R;
-  var B => A;
+  ref B = A;
```

Comment:
Let's remove it because the parent directory already has `4-return-local-ref.chpl` which covers this case.